pyerrors/input/pandas.py

```diff
@@ -27,7 +27,11 @@ def to_sql(df, table_name, db, if_exists='fail', gz=True, **kwargs):
     -------
     None
     """
-    se_df = _serialize_df(df, gz=gz)
+    se_df = _serialize_df(df, gz=False)
```

Comment:
I am still trying to understand these changes. Wouldn't it be better to include this bit into `_serialize_df` or is there a reason why the serialization is split up into two different parts of the code?